entity-framework/core/managing-schemas/migrations/applying.md

```diff
@@ -156,6 +159,76 @@ Note that this can be used to roll back to an earlier migration as well.
 
 For more information on applying migrations via the command-line tools, see the [EF Core tools reference](xref:core/cli/index).
 
+## Bundles
+
+> [!NOTE]
+> This feature was added in EF Core 6.0.
+
+Migration bundles are single-file executables than can be used to apply migrations to a database. They address some of the shortcomings of the SQL script and command-line tools:
+
+* Executing SQL scripts requires additional tools.
+* The transaction handling and continue-on-error behavior of these tools are inconsistent and sometimes unexpected.
+* Bundles can be generated as part of your CI process and executed later as part of your deployment process.
+* Bundles can be executed without installing the .NET SDK or EF Tool (or even the .NET Runtime when self-contained), and they don't require the project's source code.
+
+### [.NET Core CLI](#tab/dotnet-core-cli)
+
+The following generates a bundle:
+
+```dotnetcli
+dotnet ef migrations bundle
+```
+
+The following generates a self-contained bundle for Linux:
+
+```dotnetcli
+dotnet ef migrations bundle --self-contained -r linux-x64
+```
+
+### [Visual Studio](#tab/vs)
+
+The following generates a bundle:
+
+```powershell
+Bundle-Migration
+```
+The following generates a self-contained bundle for Linux:
```

Comment:
Given that this is a VS sample, should we show Windows instead?

Reply:
My intention is to show how to generate for a different OS.